src/c++/perf_analyzer/genai-perf/genai_perf/main.py

```diff
@@ -110,12 +111,13 @@ def report_output(data_parser: LLMProfileDataParser, args: Namespace) -> None:
     stats.export_to_csv(export_csv_name)
     stats.export_parquet(DEFAULT_PARQUET_FILE)
     stats.pretty_print()
-    create_graphs(stats)
+    create_plots(stats, args.generate_plots)
```

Comment:
Whether or not to generate plot should be decided by the caller and callee shouldn't need to care about it.
```suggestion
    if args.generate_plots:
        create_plots(stats)
```

Reply:
good call. ill update

Reply:
Updated.